set.mm

```diff
@@ -51613,6 +51695,37 @@ we define it to be empty in this case (see ~ opprc1 , ~ opprc2 , and
     ( vx cvv cpw cv wcel wss ssv selpw mpbir vex 2th eqriv ) ABCZBADZMEZNBEONBF
     NGABHIAJKL $.
 
+  ${
+    $d A v $.  $d B v $.  $d C v $.  $d V v $.
+    $( For an element of a proper unordered pair of elements of a class ` V ` ,
+       there is another (different) element of class ` V ` which is an element
```

Comment:
"of `V`" or "of the class `V`", but not "of class `V`"

Reply:
OK, I'll fix that.